package.json

```diff
@@ -87,6 +87,7 @@
     "@grafana/experimental": "^1.1.0",
     "@grafana/runtime": "9.2.1",
     "@grafana/ui": "9.2.1",
+    "@testing-library/dom": "^9.3.1",
```

Comment:
I'm not sure why this got added here? If you back out that change and re-run `yarn install --pure-lockfile` what happens?